subworkflows/local/input_check.nf

```diff
@@ -45,6 +43,43 @@ workflow INPUT_CHECK {
         .map { db_meta, db_path ->
             if (db_meta.type in ["blastp", "blastx"] && db_path.isDirectory()) {
                 [db_meta, file(db_path.toString() + "/${db_path.name}", checkIfExists: true)]
+            } else if (db_meta.type == "blastn") {
+                // Special handling for BLAST nucleotide databases
+                // If db_path is a directory (from untar), look for .nal file inside or .nin file as fallback
+                def actual_db_path = db_path
+                if (db_path.isDirectory()) {
+                    // First priority: look for .nal files
+                    def nal_files = db_path.listFiles().findAll { it.name.endsWith('.nal') }
```

Comment:
`.listFiles()` doesn't resolve symlinks. For `/data/tol/resources/nt/latest` it only returns itself instead of listing the files in `/lustre/scratch122/tol/resources/nt/2025-11-20T00:00:00`.
There must be a Groovy function to resolve symlinks, and then you can call `listFiles()` fine.